aspnetcore/blazor/file-downloads.md

```diff
@@ -20,7 +20,7 @@ Files can be downloaded from the app's own static assets or from any other locat
 
 :::moniker range=">= aspnetcore-9.0"
 
-* ASP.NET Core apps use Map Static Assets Middleware or Static File Middleware to serve files to clients of server-side apps. For more information, see <xref:blazor/fundamentals/static-files>.
+* ASP.NET Core apps use Map Static Assets routing endpoint conventions or Static File Middleware to serve files to clients of server-side apps. For more information, see <xref:blazor/fundamentals/static-files>.
```

Comment:
This level of detail seems unnecessary here. Does the reader really need to know that there are two ways to serve static files from an ASP.NET Core app (an older one and a newer one) when learning how to download a file? Consider removing these two bullet points and just making the "static assets" text above the link to the static files article.